src/avram/associations/has_many.cr

```diff
@@ -1,7 +1,10 @@
 module Avram::Associations::HasMany
   macro has_many(type_declaration, through = nil, foreign_key = nil)
-    {% if !through.is_a?(NilLiteral) && !through.is_a?(SymbolLiteral) %}
-      {% through.raise "The association name for 'through' must be a Symbol. Instead, got: #{through}" %}
+    {% if !through.is_a?(NilLiteral) && (!through.is_a?(ArrayLiteral) || through.any? { |item| !item.is_a?(SymbolLiteral) }) %}
+      {% through.raise "'through' must be given an Array of Symbols. Instead, got: #{through}" %}
```

Comment:
What about an error like this?

```crystal
      {% through.raise <<-ERROR
      'through' on #{@type.name} must be given an Array of Symbols. Instead, got: #{through}
      
      Example...
         has_many comments : Comment
         has_many related_authors : User, through: [:comments, :author]
      
      Learn more about associations: https://luckyframework.org/guides/database/models#model-associations
      ERROR
      %}
```

We could also see if it's possible to put the exact file and such like we've done in some other errors. I'm just thinking "what goes in this array?" type deals.